spec/parsers/pypi_spec.rb

```diff
@@ -204,6 +204,34 @@
     })
   end
 
+  it "parses dependencies from pip-dependency-tree.json" do
```

Comment:
🎱 Should there be a couple of cases to test out the `.uniq` calls?

Reply:
done-so